src/item.cpp

```diff
@@ -3443,16 +3444,19 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
         const item &contents_item = contents.front();
         if( contents_item.made_of( LIQUID ) || contents_item.is_food() ) {
             const unsigned contents_count = contents_item.charges > 1 ? contents_item.charges : quantity;
-            maintext = string_format( pgettext( "item name", "%s of %s" ), label( quantity ),
+            //~ %1$s: item name, %2$s: content liquid, food, or drink name
+            maintext = string_format( pgettext( "item name", "%1$s of %2$s" ), label( quantity ),
                                       contents_item.tname( contents_count, with_prefix ) );
         } else {
-            maintext = string_format( pgettext( "item name", "%s with %s" ), label( quantity ),
+            //~ %1$s: item name, %2$s: non-liquid, non-food, non-drink content item name
+            maintext = string_format( pgettext( "item name", "%1$s with %2$s" ), label( quantity ),
                                       contents_item.tname( quantity, with_prefix ) );
         }
     } else if( !contents.empty() ) {
+        //~ %1$s: item name, %2$zd: content size
         maintext = string_format( npgettext( "item name",
-                                             "%s with %zd item",
-                                             "%s with %zd items", contents.size() ),
+                                             "%1$s with %2$zd item",
```

Comment:
A non-adjacent comment will not be extracted for this translation.

Reply:
I think it checks for the npgettext call instead of the actual string?

Reply:
It doesn't seem like that.

Updated pot fragment for the line in my next comment:
```pot

#: src/iuse_actor.cpp
#, c-format
msgctxt "healing"
msgid "Select a body part of %1$s for %2$s:"
msgstr ""
```